Integrations/FidelisElevateNetwork/FidelisElevateNetwork.py

```diff
@@ -177,6 +186,346 @@ def generate_time_settings(time_frame=None, start_time=None, end_time=None):
+    url = '/j/rest/v1/alert/mgmt/'
+    http_request('PUT', url, data=data)
+
+
+def manage_alert_label_command():
+    args = demisto.args()
+    alert_id = args['alert_id']
+    label = args['label']
+    action = args['action']
+
+    label_action = {
+        'Add': 'LABEL_ADD',
+        'Remove': 'LABEL_REMOVE'
+    }
+
+    data = {
+        'type': "byAlertID",
+        'alertIds': [alert_id],
+        'labels': [label],
+        'labelAction': label_action[action],
+    }
+
+    bad_res = manage_alert_label(data)
+    if bad_res and action == 'Add':
+        return_error("Was not able to add the label {} to alert {}".format(label, alert_id))
+
+    elif bad_res and action == 'Remove':
+        return_error("Was not able to remove the label {} to alert {}".format(label, alert_id))
+
+    else:
+        return_outputs("Assigned label: {} to alert {}".format(label, alert_id), {}, {})
+
+
+@logger
+def manage_alert_label(data):
+    url = '/j/rest/v1/alert/mgmt/'
+    res = http_request('PUT', url, data=data)
+    if res.get('Console') == 'OK':
+        return 0
+
+    else:
+        return 1
+
+
+def manage_alert_assignuser_command():
+    args = demisto.args()
+    conclusion_id = args['conclusion_id']
+    assign_user = args['assign_user']
+    comment = args.get('comment')
+
+    data = {
+        'alertIds': ['Console-' + str(conclusion_id)],
+        'assignToUser': assign_user,
+        'searchParams': None,
+        'byId': True,
+        'purgeEvents': False,
+        'resolution': None,
+        'comment': comment,
+        'labels': None,
+        'rating': None,
+        'status': 'OPEN',
+        'action': "ASSIGN",
+    }
+
+    raw_response = manage_alert_assignuser(data)
+    entry_context = {
+        'AssignedUser': assign_user,
+        'ConclusionID': conclusion_id
+    }
+
+    return_outputs("Assigned User: {} to alert with conclusion ID {}".format(assign_user, conclusion_id),
+                   {'Fidelis.Alert(val.ConclusionID && val.ID == obj.ConclusionID)': entry_context}, raw_response)
+
+
+def manage_alert_assignuser(data):
+    url = '/j/rest/v2/alert/mgmt/'
+    raw_res = http_request('POST', url, data=data)
+    return raw_res
+
+
+def manage_alert_closealert_command():
+    args = demisto.args()
+    conclusion_id = args['conclusion_id']
+    comment = args.get('comment')
+    resolution = args['resolution']
+
+    data = {
+        'alertIds': ['Console-' + str(conclusion_id)],
+        # This field is not used by Fidelis when closing alerts / So setting it doesn't matter
+        'searchParams': None,
+        'byId': True,
+        'purgeEvents': False,
+        'resolution': resolution,
+        'comment': comment,
+        'labels': None,
+        'rating': None,
+        'status': 'CLOSED',
+        'action': "STATUS",
+    }
+
+    raw_response = manage_alert_closealert(data)
+
+    return_outputs("Closed alert conclusion ID {}".format(conclusion_id), {}, raw_response)
```

Comment:
@roysagi We might want to add to the context that the conclusion was closed, what do you think?

Reply:
Thing is the Conclusion ID is no the same as the Alert ID - and the Alert ID is not returned by this command and the conclusion ID is not brought back by other commands... so there is no way to connect the context of the existing alert.